RedHat-UX/red-hat-design-system
# docs(pagination): update docs

Closes #930

## What I did

1. Updated the documentation for the pagination element according to the specs outlined in #930.


## Testing Instructions

1. Review latest docs at https://deploy-preview-993--red-hat-design-system.netlify.app/elements/pagination/ and verify they match the design specs as well as the docs cleanup guide https://github.com/RedHat-UX/red-hat-design-system/wiki/Docs-Cleanup-Q2-2023-Checklist#code-page.
